assets/scripts/streets/StreetName.jsx

```diff
@@ -116,6 +103,7 @@ export default class StreetName extends React.Component {
 }
 
 StreetName.propTypes = {
+  id: React.PropTypes.string,
   allowEditing: React.PropTypes.bool,
   street: React.PropTypes.any,
   handleResize: React.PropTypes.func
```

Comment:
Now that `StreetName` is used in more places (:+1:) the `handleResize` prop is not always passed in, so `this.props.handleResize()` can be called but throws an error if it's not present. I'd recommend passing in a no-op function as a default prop for `handleResize` but let me know what you think.